### DISCUSSION TITLE
FastAPI 0.100.0 not working with Pydantic 1.9.2

### DISCUSSION THREAD

**Discussion by MFAlexandru** [fastapi/fastapi#9874] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import fastapi
```

### Description

When using the mentioned versions fastapi cannot be imported because `MultipleHostDsn` is missing from `pydantic.networks`.
I see that fastapi has some restrictions for the pydantic versions but 1.9.2 is not there, maybe it should be included.

### Operating System

macOS

### FastAPI Version

0.100.0

### Python Version

3.10.0

**Comment by Kludex** [comment, marked as answer]
Yeah. We have a PR for it: https://github.com/tiangolo/fastapi/pull/9852

**Comment by freddyaboulton** [comment]
Nice to see a PR! ran into this here https://github.com/gradio-app/gradio/issues/4912 with version 1.9.0
